pkg/local_object_storage/engine/engine.go

```diff
@@ -33,6 +34,11 @@ type StorageEngine struct {
 	}
 }
 
+// ContainersSource must return actual container list in the map form.
```

Comment:
interface is defined in the form of a requirement - looks very strange. I'd say
```
// ContainerSource is a source of information about NeoFS containers.
```
and
```
// GetContainers returns a list of containers for which data is subject to further storage.
```